pkg/client/clientset_generated/internalclientset/typed/core/unversioned/fake/fake_node_expansion.go

```diff
@@ -23,7 +23,7 @@ import (
 
 func (c *FakeNodes) PatchStatus(nodeName string, data []byte) (*api.Node, error) {
 	obj, err := c.Fake.Invokes(
-		core.NewPatchSubresourceAction(nodesResource, "status"), &api.Node{})
```

Comment:
oh, ok. Because it was in a commit called "generated changes", I assumed they are auto generated. make sense to me know. The change looks good in respect to actual code change, I am still trying to understand the whole client set, but no need to block this PR for that.